docs/stac-api-spec/extensions/layer/README.md

```diff
@@ -0,0 +1,141 @@
+# STAC API Layer Extension
+
+**Extension [Maturity Classification](https://github.com/radiantearth/stac-api-spec/blob/master/extensions/README.md#extension-maturity): Proposal**
+
+This API extension defines the API resources and semantics for retrieving [STAC Layers](../../../stac-spec/extensions/layer/README.md)
+information defined for available STAC Items.
+
+## Methods
+
+| Path                    | Content-Type Header                                                                                         | Description                                                                       |
+| ----------------------- | ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------- |
+| `GET /layers/`          | [Catalog](https://github.com/radiantearth/stac-api-spec/blob/master/stac-spec/catalog-spec/catalog-spec.md) | Catalog with links to layers.                                                      |
+| `GET /layers/{layerID}` | [`GeoJSON Feature`](https://github.com/azavea/stac4s/blob/4aba9c6691fde1d5235f165454ceaef6c6b3e165/docs/stac-spec/extensions/layer/json-schema/layer-schema.json)                                                                                       | [STAC Layer](../../../stac-spec/extensions/layer/README.md) as a GeoJSON Feature. |
```

Comment:
This provides the `layer` as a `Feature`... is there an additional endpoint for items in the layer as a `FeatureCollection`? To mirror the `/collections/<collection-id>/items` endpoint. Or is it assumed that people will use the query extension to find those items?

Reply:
It is assumed the the STAC query extension should be used. I probably need to mention that, a very good point!